src/stores/networkStore.ts

```diff
@@ -70,12 +70,19 @@ export class NetworkStore {
         this._isPointsVisible = !this._isPointsVisible;
     }
 
+    @action
+    public setSelectedTransitTypes(types: TransitType[]) {
+        this._selectedTransitTypes = types;
+    }
+
+    // TODO rename as toggleSelectedTransitType?
     @action
     public toggleTransitType(type: TransitType) {
         if (this._selectedTransitTypes.includes(type)) {
             this._selectedTransitTypes = this._selectedTransitTypes.filter(t => t !== type);
         } else {
-            this._selectedTransitTypes.push(type);
+            // Need to do concat (instead of push) to trigger ReactionDisposer watcher
```

Comment:
It's good to change from push to concat, but we can maybe remove the comment? maybe

Reply:
I think that comment is fine so that it tries to prevent changing this code line to use .push()